mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/meta/WildcardManager.java

```diff
@@ -99,9 +99,7 @@ private void refreshWildcards(String firstTopic) {
             queueNames.add(pubTopic);
             Set<String> wildcards = matchWildcards(pubTopic);
             if (wildcards != null && !wildcards.isEmpty()) {
```

Comment:
line 101 the return wildcards may never be null and the if judgment can be removed or not ？

Reply:
I prefer not to removed. It's not expensive and could avoid problems that may arise from a change of matchWildcards()  in the future.

Reply:
agree that